docs/runbooks/VirtHandlerRESTErrorsBurst.md

```diff
@@ -2,11 +2,9 @@
 
 ## Meaning
 
-For the last 10 minutes or longer, over 80% of the REST calls made to
-`virt-handler` pods have failed.
-
-This alert usually indicates that the `virt-handler` pods cannot connect to the
-API server.
+More than 80% of REST calls failed in `virt-handler` in the last 5 minutes.
+This alert usually indicates that the `virt-handler` pods cannot connect
+to the API server.
```

Comment:
Same comment as on the `VirtApiRESTErrorsBurst.md` file

Reply:
Same

Reply:
https://github.com/kubevirt/monitoring/pull/240#discussion_r1590437383